git-branchless-hook/src/lib.rs

```diff
@@ -42,6 +42,41 @@ pub use lib::core::rewrite::rewrite_hooks::{
     hook_skip_upstream_applied_commit,
 };
 
+/// A component that will find the hash for a given reference transaction OID.
+/// Reference transaction OIDs may be one of:
+///
+/// * An up-to-40-hex-digit hash string
+/// * A symbolic ref, in which case, it will start with "ref:"
+///
+/// See: https://github.com/git/git/commit/a8ae923f85da6434c3faf9c39719d6d5e5c77e65
```

Comment:
```suggestion
/// See: "refs: support symrefs in 'reference-transaction' hook" https://github.com/git/git/commit/a8ae923f85da6434c3faf9c39719d6d5e5c77e65
```

suggestion: Add the commit title here somewhere so that the reader can more easily decide if they want to click the link (I don't feel strongly about the specific formatting).

comment (non-blocking):

> We do not add any tests in this commit since there is no git command which activates this flow

great thank you very much Git developers